pandas/core/internals/blocks.py

```diff
@@ -224,14 +224,6 @@ def get_values(self, dtype: DtypeObj | None = None) -> np.ndarray:
         # expected "ndarray")
         return self.values  # type: ignore[return-value]
 
-    @final
-    def get_block_values_for_json(self) -> np.ndarray:
```

Comment:
i think the "final" is out of date; same method on DatetimeLikeBlock

Reply:
There is only a single `get_block_values_for_json` (no Block subclass overrides it)

Reply:
Or at least until a few hours ago, when your PR that changed this was merged. You could have just as well pointed to https://github.com/pandas-dev/pandas/pull/41082